docs/sphinx/examples/python/tutorials/quantum_bits.ipynb

```diff
@@ -175,6 +175,215 @@
     "\n"
    ]
   },
+  {
```

Comment:
This format is difficult to parse. Hence adding review comments in on this file one place here.

- You can install `QuTip` in this notebook. Add a cell `!pip install qutip>=5` at the beginning.
- The error message is not getting displayed if QuTip is no installed, instead following error shows up
```TypeError: install_qutip_request() takes 0 positional arguments but 1 was given```

Reply:
Fixed in [a3964e2](https://github.com/NVIDIA/cuda-quantum/pull/1763/commits/a3964e284910f99cbbddddfa7493747ab9735855). I've added a line to install `QuTiP` in the notebook, and also fixed the function to display the error each time if `QuTiP` is not installed. Currently the function simply asks to install `QuTiP` each time the visualization code is called, but it doesn't exit the kernel by throwing an exception. This is by design, so the user doesn't lose progress and the kernel doesn't accidentally die of a thrown exception. So if this is done in a loop, it will show multiple errors.

Reply:
I tried out this notebook locally, even after executing the cell which install QuTip, I'm getting following error -
```The module QuTiP was not detected. Bloch Sphere visualization will not work.
Please install QuTip to use this feature. You can run "pip install qutip" or in a conda environment, run "conda -c conda-forge install qutip" to install qutip. 
```

Reply:
Hi, you need to restart the notebook kernel after executing the install statement, else it will not load the packages. This is unfortunately a limitation of Jupyter. I've added a comment in the code informing the user to do this as well.

Reply:
CI failures owing to Jupyter notebook not auto-reloading.